phy/io/kwik/store_items.py

```diff
@@ -528,8 +528,14 @@ def n_unmasked_channels(self, cluster):
 
     def mean_probe_position(self, cluster):
         mean_masks = self.load(cluster, 'mean_masks')
-        return _mean(self.model.probe.positions *
-                     mean_masks[:, np.newaxis], (2,))
+        if mean_masks is not None and mean_masks.shape[0]:
+            meanClusterPosition = (np.sum(self.model.probe.positions *
```

Comment:
could you rename this to `mean_cluster_position`? This is to stay consistent with the rest of the code and with standard Python conventions